# Confidence Interval

## given data = 1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 
## 1.48, 1.20, 1.33, 1.18, 1.22, 1.29
- Sample size (n) = 15
- Confidence level = 99%

In [1]:
# Mean (x̄) Calculate
import numpy as np

data = [1.13,1.55,1.43,0.92,1.25,1.36,1.32,0.85,1.07,
        1.48,1.20,1.33,1.18,1.22,1.29]

mean = np.mean(data)
mean

1.2386666666666666

In [2]:
# Standard Deviation (s)
std = np.std(data, ddof=1)
std

0.19316412956959936

In [3]:
# t-value Find (99%, df = 14)
# in table = t ≈ 2.977

In [4]:
# Calculation
import math

n = 15
t = 2.977

margin = t * (std / math.sqrt(n))

lower = mean - margin
upper = mean + margin

lower, upper

(1.0901894948523831, 1.38714383848095)

In [5]:
# Case 2: Population SD Known (σ = 0.2)
# Z value (99%)
# Z ≈ 2.576

In [6]:
# Calculation
sigma = 0.2
z = 2.576

margin = z * (sigma / math.sqrt(n))

lower = mean - margin
upper = mean + margin

lower, upper

(1.105642598668929, 1.371690734664404)

# Hypothesis Testing

## Given:
- Sample mean = 3050
- n = 25
- X mean = 600
- σ = 5 × 25 = 125

# Theoretical Mean (μ)

### Formula:

W = 1000 + 5X

Put X = 600:

μ = 1000 + (5 × 600)
μ = 4000

### Hypotheses
- H₀ (Null): μ = 4000
- H₁ (Alt): μ > 4000

In [9]:
# Calculation
x_bar = 3050
mu = 4000
sigma = 125
n = 25

z = (x_bar - mu) / (sigma / math.sqrt(n))
z

-38.0

Critical Value

For α = 0.05 (one-tailed):

Z critical = 1.645

## Z = -38  < 1.645

## Confidence Interval:
- Mean calculate 
- t/z use
- Result: (1.09 , 1.39)
## Hypothesis Testing:
- μ = 4000 
- Z-test apply 
- Conclusion: Not Claim supported 